### DISCUSSION TITLE
Can't delete accounts as an admin in hubs cloud

### DISCUSSION THREAD

**Discussion by kala-goyal** [mozilla/hubs#2747] [General Questions & Discussion] [answered]
I have an instance of Hubs Cloud running. I've been adding accounts from my end to make sure the events we'll hold with the hubs stay safe. I was testing some functionalities, and I noticed that I couldn't delete accounts once created. When I go to Accounts->click on account->Delete, it seems to delete but after about a second, if I click on the screen it seems to glitch and refresh the page, and the account is back.
Here's a 15 second screen recording of what happens-
https://youtu.be/hy4EDQ754kA

**Comment by robinkwilson** [comment, marked as answer]
Deletion in the admin panel is not implemented yet (See https://github.com/mozilla/hubs/issues/2647)

Currently, for Hubs Cloud you need to "disable" accounts. Here's new documentation on [HC Account Management and Limiting Access to your instance](https://hubs.mozilla.com/docs/hubs-cloud-limiting-user-access.html).

HC instance > Admin panel > Accounts > Click "Edit" button for the account you want to disable (or use find by email address) > Select "disable" in dropdown menu

**Reply by kala-goyal** [reply]
is there any other way to delete accounts for right now then? Like some command to do so from the console or in the S3 bucket? Or to reset accounts without having to create an entirely new instance

**Reply by robinkwilson** [reply]
No just the "disabling" currently.

**Reply by ConstantChange** [reply]
But @robin-k-wilson what if I have 25 pages of accounts, with 25 entries per page? We need a way to batch disable
